book/core-concepts/windows.md

```diff
@@ -0,0 +1,152 @@
+# Windows
+
+Wallaroo supports windowing over aggregations. This allows you to break an infinite stream into manageable chunks and also allows you to see how your inputs evolve over time.
+
+There are two broad categories of windows currently supported: count-based and range-based. Range-based windows can be further divided into tumbling and sliding varieties. We'll look at each of these in turn.
+
+If you are not familiar with aggregations in Wallaroo, it might be helpful to read [this section](/book/core-concepts/aggregations.md) before proceeding.
+
+## Count-based Windows
+
+Count-based windows emit an output every `n` input messages, where `n` is specified via the API. For example (using the Python API):
+
+```python
+    (inputs
+        .to(wallaroo.count_windows(5)
+            .over(MySumAgg))
```

Comment:
The `aggregations.md` doc called it `Sum`. Should one of them change to match the other?

Reply:
addressed